### PR TITLE
Retry when rpc failure occurs

### PR DESCRIPTION
**Pull request by DragonDriver**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>

### CODE REVIEW

File: pymilvus/client/stub.py [hunk 1]
```diff
@@ -41,6 +42,30 @@ def inner(self, *args, **kwargs):
...
 
+def retry_on_rpc_failure(retry_times=10, wait=1):
+    def wrapper(func):
+        @functools.wraps(func)
+        def handler(self, *args, **kwargs):
+            counter = 1
+            try:
+                return func(self, *args, **kwargs)
+            except grpc.RpcError as e:
+                if e.code() != grpc.StatusCode.DEADLINE_EXCEEDED and e.code() != grpc.StatusCode.UNAVAILABLE:
```
Comment (scsven) [inline pymilvus/client/stub.py:53]: If the exception is DEADLINE_EXCEEDED or UNAVAILABLE, these code behavior? 
I think this decorator need some comments to explain behavior.

**Review by scsven — CHANGES_REQUESTED** [review]

File: pymilvus/client/stub.py (hunk 1 above)
Comment (DragonDriver) [reply pymilvus/client/stub.py:53]: According to the [doc of grpc](https://grpc.github.io/grpc/python/grpc.html#grpc-status-code), DEADLINE_EXCEEDED means the task was not completed, UNAVAILABLE means that service is not reachable currently.

**Pull request merged**